### ISSUE TITLE
Configure Fields

### ISSUE THREAD

**Issue by timscott** [kovndr/braintree-dropin-react#1]
I noticed that the credit card form does not have the CVV. I'm new to Braintree, but I think this can be by passing `options.card` into [`create`](https://braintree.github.io/braintree-web-drop-in/docs/current/module-braintree-web-drop-in.html#.create). That would seem to be a simple extension of your component. Are you open to pull requests for that (and possibly other customizations)?

**Comment by kovndr** [comment]
Go to braintree website/settings/processing/CVV and check option **CVV not provided (I)** with **For Any Transaction**
Btw an update is on the way however thanks for taking the time to contribute! ;)

**Comment by timscott** [comment]
Thanks. Yeah, I figured that out after playing with the code locally a bit. I don't have a reason then to add `card` prop. However, I might still be submitting a PR. I would like the component to accept `className` so it will play with [styled-components](https://github.com/styled-components/styled-components).

One small thing I also noticed. You are using `selector`. The docs say it's been deprecated in favor of `container`.

(By the way, thanks for building this. This is my first time using Braintree in over 5 years. I been using Stripe. Seems Braintree has come a long way, and it's nice to have a react component to use. I'm not sure if I will stay with the Drop In, but it's great to get it up and running fast.)
